{
  "comments": [
    {
      "unresolved": true,
      "key": {
        "uuid": "11c2d9c0_0dac0918",
        "filename": "src/restconf/uri.cpp",
        "patchSetId": 2
      },
      "lineNbr": 200,
      "author": {
        "id": 1000000
      },
      "writtenOn": "2023-12-18T16:41:11Z",
      "side": 1,
      "message": "This looks like a wrong error message",
      "range": {
        "startLine": 200,
        "startChar": 68,
        "endLine": 200,
        "endChar": 96
      },
      "revId": "cfbcc1d3eb1fd51b149606bbf631a5db34a61b13",
      "serverId": "e32f8df8-3db0-4f76-a6c6-fe9d6e69dd68"
    },
    {
      "unresolved": true,
      "key": {
        "uuid": "8d655959_1492d680",
        "filename": "src/restconf/uri.cpp",
        "patchSetId": 2
      },
      "lineNbr": 225,
      "author": {
        "id": 1000000
      },
      "writtenOn": "2023-12-18T16:41:11Z",
      "side": 1,
      "message": "doxygen\u0027s grammar says that this tag expects the exception type name followed by a description on when it\u0027s thrown",
      "range": {
        "startLine": 225,
        "startChar": 3,
        "endLine": 225,
        "endChar": 59
      },
      "revId": "cfbcc1d3eb1fd51b149606bbf631a5db34a61b13",
      "serverId": "e32f8df8-3db0-4f76-a6c6-fe9d6e69dd68"
    },
    {
      "unresolved": false,
      "key": {
        "uuid": "0982b456_b7026120",
        "filename": "src/restconf/uri.cpp",
        "patchSetId": 2
      },
      "lineNbr": 228,
      "author": {
        "id": 1000000
      },
      "writtenOn": "2023-12-18T16:41:11Z",
      "side": 1,
      "message": "OK, this makes sense after reading the next patch in the series.",
      "revId": "cfbcc1d3eb1fd51b149606bbf631a5db34a61b13",
      "serverId": "e32f8df8-3db0-4f76-a6c6-fe9d6e69dd68"
    }
  ]
}